frontend/src/component/changeRequest/ChangeRequest/Changes/Change/StrategyChange.tsx

```diff
@@ -267,6 +267,56 @@ const UpdateStrategy: FC<{
     );
 };
 
+const AddStrategy: FC<{
+    change: IChangeRequestAddStrategy;
+    actions?: ReactNode;
+}> = ({ change, actions }) => {
+    return (
+        <>
+            <ChangeItemCreateEditDeleteWrapper>
+                <ChangeItemInfo>
+                    <Typography
+                        color={
+                            change.payload?.disabled
+                                ? 'action.disabled'
+                                : 'success.dark'
+                        }
+                    >
+                        + Adding strategy:
+                    </Typography>
+                    <StrategyTooltipLink
+                        name={change.payload.name}
+                        title={change.payload.title}
+                    >
+                        <StrategyDiff
+                            change={change}
+                            currentStrategy={undefined}
```

Comment:
it's much more obvious when we pass undefined as currentStrategy for the AddStrategy component. You don't have to analyze useCurrentStrategy why it returns undefined